SCons/Tool/MSCommon/__init__.py

```diff
@@ -30,20 +30,23 @@
 import SCons.Platform.win32
 import SCons.Util
 
-from SCons.Tool.MSCommon.sdk import mssdk_exists, \
-                                    mssdk_setup_env
+from SCons.Tool.MSCommon.sdk import mssdk_exists, mssdk_setup_env
```

Comment:
> __[Flake8]__ 'SCons.Tool.MSCommon.sdk.mssdk_exists' imported but unused ([view](https://sider.review/gh/repos/104670160/pulls/4080#issue-6441659))
>
> <details>
>
> | Rule |
> | ---- |
> | `F401` |
>
> You can [close this issue](https://sider.review/gh/repos/104670160/pulls/4080#issue-6441659) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).
>
> </details>

Okay, sider is complaining about these imports (which the PR "touches" by way of reformatting, but there is no addition/deletion).  Is it right that these aren't actually needed here?  I presume there's some purpose why they were originally put in.  If we want to keep I can close these through sider do it doesn't complain further.

Reply:
They are used by the mssdk tool

```
SCons/Tool/mssdk.py:from .MSCommon import mssdk_exists, \
SCons/Tool/mssdk.py:                     mssdk_setup_env
```

Reply:
yeah, okay, but is there any value to it importing it indirectly like that?  why not `from .sdk import mssdk_exists`?  I guess it's a little less obviously coupled, but still...

Reply:
I've seen such in other codebases.. one place import a modules exported symbols from.. and don't reach "into it"?

Reply:
yeah... it didn't immediately pop out that mssdk module is on the same level as MSCommon, not inside it.  Now I'm more curious - this kind of usage shouldn't "fool" sider, which is looking at the whole project, not at a single file, and thus should know those symbols are imported by someone else, not truly unused.

Reply:
Indeed.. odd.